code-monkey/src/tools/utils.py

```diff
@@ -16,14 +17,13 @@ def ask_user(prompt: str) -> str:
 
 
 def show_diff(original_file: str, modified_file: str) -> str:
+    print(f"Diffing {original_file} and {modified_file}")
```

Comment:
rm?

Reply:
actually going to put that in a debug log.  it was useful to figure out what was broken with file paths post-rebase.